easybuild/easyconfigs/f/FMRIprep/FMRIprep-1.4.1-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,137 @@
+    }),
+    ('MarkupSafe', '1.1.1', {
+        'checksums': ['29872e92839765e546828bb7754a68c418d927cd064fd4708fab9fe9c8bb116b'],
+    }),
+    ('Jinja2', '2.10.1', {
+        'checksums': ['065c4f02ebe7f7cf559e49ee5a95fb800a9e4528727aec6f24402a5374c65013'],
+    }),
+    ('tqdm', '4.33.0', {
+        'checksums': ['1dc82f87a8726602fa7177a091b5e8691d6523138a8f7acd08e58088f51e389f'],
+    }),
+    ('prov', '1.5.3', {
+        'checksums': ['a0c951107eafd508d5a26caa54863dc11fee3da86778db2a559d32450d2030a9'],
+    }),
+    ('neurdflib', '5.0.0.post1', {
+        'modulename': 'rdflib',
+        'checksums': ['65ef138f7a20f646b5257d312e14c624a83abdac4df234e5edcc77dc63bef5e1'],
+    }),
+    ('num2words', '0.5.10', {
+        'checksums': ['37cd4f60678f7e1045cdc3adf6acf93c8b41bf732da860f97d301f04e611cc57'],
+    }),
+    ('bids-validator', '1.2.4', {
+        'checksums': ['cf456428219c7985c13e462380bff31453c6694532f67ca47fd89b19620d77d7'],
+    }),
+    ('SQLAlchemy', '1.3.6', {
+        'checksums': ['217e7fc52199a05851eee9b6a0883190743c4fb9c8ac4313ccfceaffd852b0ff'],
+    }),
+    ('nipype', '1.2.0', {
+        'checksums': ['a5b5afbfec522686a0ac7826b6cfd82496f4ef48f6c48afb602225b0aa7b5f25'],
+    }),
+    ('nitime', '0.8.1', {
+        'checksums': ['269eae522f613c9ca8c3d62b66074a76b40d9484f468b4b8525f7c9342e96341'],
+    }),
+    ('grabbit', '0.2.6', {
+        'checksums': ['134fdad2d079693f8836f1074241d01ba5ed38f768c26279a3fe39afb6757f5b'],
+    }),
+    ('lockfile', '0.12.2', {
+        'checksums': ['6aed02de03cba24efabcd600b30540140634fc06cfa603822d508d5361e9f799'],
+    }),
+    ('appdirs', '1.4.3', {
+        'checksums': ['9e5896d1372858f8dd3344faf4e5014d21849c756c8d5701f78f8a103b372d92'],
+    }),
+    ('rply', '0.7.7', {
+        'checksums': ['4d6d25703efd28fb3d5707f7b3bd4fe66c306159a5c25af10ba26d206a66d00d'],
+    }),
+    ('citeproc-py', '0.4.0', {
+        'modulename': 'citeproc',
+        'checksums': ['ed513dbc76f782b5e98126d6bebbd1284841fcf199ec9dda552e2bce864adadf'],
+    }),
+    ('duecredit', '0.7.0', {
+        'checksums': ['8a13be317afcf56bdccba6a5d3d82c2f4e3509ffa1aff47d61c1a91e6d454010'],
+    }),
+    ('tedana', '0.0.7', {
+        'checksums': ['9f541bdf9ecd06b883e3bb2a753987279aabe7b7a55651a59916a798a6578c81'],
+    }),
+    ('pybids', '0.7.1', {
+        'modulename': 'bids',
+        'checksums': ['a4725f691e9fa05294338b049d2babc623a6feb718d2607a8c56146537e309b4'],
+    }),
+    ('templateflow', '0.3.0', {
+        'checksums': ['14830702c293e870a69fdefca2928618179cdbb631ce84a99522aa859a1f4205'],
+    }),
+    ('niworkflows', '0.9.6', {
+        'patches': ['niworkflows-%(version)s_fix-bus-error.patch'],
+        'checksums': [
+            '07275844f5e830562109acca7340f971b98e4a22e7938d8d5aceb2cb5891049c',  # niworkflows-0.9.6.tar.gz
+            'd42ea11f54b4be75e68764a3bb74e6d202e60e4e6b174ded217fe18acb439133',  # niworkflows-0.9.6_fix-bus-error.patch
```

Comment:
@boegel this patch version is picking up the main package version

Reply:
@verdurin Are you saying `%(version)s` is being resolved with `1.4.1` here?
Can you clarify in what situation exactly? Which errors are you hitting?

Reply:
@boegel Yes, this is what Travis is complaining about.